hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/index/SqlIndexConverterMismatchTest.java

```diff
@@ -93,7 +95,11 @@ public void before() {
         map = member1.getMap(MAP_NAME);
     }
 
+    /**
+     * @see MapIndexScanExecIterator#getIndexEntries
+     */
     @SuppressWarnings("StatementWithEmptyBody")
+    @Ignore // TODO: implement this check in operation, see the comment above.
```

Comment:
Forgotten TODO?

Reply:
Nope, it should be implemented in separate PR. We want to have such an exception, but it is still not implemented in `MapFetchIndexOperation`. In a comment above I provided a link with same check in IMDG engine.

Reply:
https://github.com/hazelcast/hazelcast/issues/19287